core/miner/impl/miner_impl.cpp

```diff
@@ -78,7 +85,39 @@ namespace fc::miner {
             api,
             deadline_info.period_start % kWPoStProvingPeriod,
             kMaxSectorExpirationExtension - 2 * kWPoStProvingPeriod);
-
+    std::shared_ptr<FeeConfig> fee_config = std::make_shared<FeeConfig>();
+    fee_config->max_precommit_batch_gas_fee.base = {
+        0};  // TODO: config loading;
+    fee_config->max_precommit_batch_gas_fee.per_sector =
+        static_cast<TokenAmount>(0.02 * 10e18);
```

Comment:
```suggestion
        TokenAmount{"200000000000000000"};
```

Reply:
Did you mean 10e18 (0.2e18) or 1e18 (0.02e18)?

Reply:
Same below

Reply:
fixed